protocol/v2/ssv/validator/committee_queue.go

```diff
@@ -99,50 +101,53 @@ func (v *Committee) ConsumeQueue(logger *zap.Logger, slot phase0.Slot, handler M
 		// Construct a representation of the current state.
 		state := *q.queueState
 
-		//runner := v.Runners.DutyRunnerForMsgID(msgID)
-		//if runner == nil {
-		//	return fmt.Errorf("could not get duty runner for msg ID %v", msgID)
-		//}
-		//var runningInstance *instance.Instance
-		//if runner.HasRunningDuty() {
-		//	runningInstance = runner.GetBaseRunner().State.RunningInstance
-		//	if runningInstance != nil {
-		//		decided, _ := runningInstance.IsDecided()
-		//		state.HasRunningInstance = !decided
-		//	}
-		//}
-		//state.Height = v.GetLastHeight(msgID)
-		//state.Round = v.GetLastRound(msgID)
-		//state.Quorum = v.Share.Quorum
-		//
-		//filter := queue.FilterAny
-		//if !runner.HasRunningDuty() {
-		//	// If no duty is running, pop only ExecuteDuty messages.
-		//	filter = func(m *queue.DecodedSSVMessage) bool {
-		//		e, ok := m.Body.(*types.EventMsg)
-		//		if !ok {
-		//			return false
-		//		}
-		//		return e.Type == types.ExecuteDuty
-		//	}
-		//} else if runningInstance != nil && runningInstance.State.ProposalAcceptedForCurrentRound == nil {
-		//	// If no proposal was accepted for the current round, skip prepare & commit messages
-		//	// for the current height and round.
-		// filter := func(m *queue.DecodedSSVMessage) bool {
-		// 	sm, ok := m.Body.(*specqbft.Message)
-		// 	if !ok {
-		// 		return true
-		// 	}
-
-		// 	if sm.Height != state.Height || sm.Round != state.Round {
-		// 		return true
-		// 	}
-		// 	return sm.MsgType != specqbft.PrepareMsgType && sm.MsgType != specqbft.CommitMsgType
-		// }
+		v.mtx.Lock()
+		runner := v.Runners[slot]
+		v.mtx.Unlock()
```

Comment:
There is no point in getting this from the map with the mutex each iteration of the loop. either pass it down from `StartDuty` into `ConsumeQueue` (better solution imo) or get it once before the `for` loop.